### PR TITLE
replace s2i with dockerfile

### PR DESCRIPTION
**Pull request by ggosiang**
Signed-off-by: Eric Yang <ericy@infuseai.io>

### CODE REVIEW

File: docs/model-deployment-language-wrapper-python.md [hunk 1]
```diff
@@ -74,61 +76,59 @@ s2i usage seldonio/seldon-core-s2i-python3:0.18
...
 
 ## Build the Image
 
-Make sure you are in the folder that includes `requirements.txt`, `.s2i/environment`, `python file for model deployment`, and `model file`...etc.
-
-If this folder is managed by `Git`, please commit all changes into the git.
+- Make sure you are in the folder that includes `requirements.txt`, `Dockerfile`, `python file for model deployment`, and `model file`.
 
-We will use `seldonio/seldon-core-s2i-python3:0.18` as a base image, installing environment and packaging our model file into the target, `my-model-image`. You can use the following command to package the docker image:
+- If this folder is managed by `Git`, please commit all changes into the git.
```
Comment (jackklpan) [inline docs/model-deployment-language-wrapper-python.md:89]: I think if we are using docker, this is not required.

File: docs/model-deployment-language-wrapper-r.md [hunk 2]
```diff
@@ -63,63 +79,61 @@ s2i usage seldonio/seldon-core-s2i-r:0.1
...
 
 ## Build the Image
 
-Make sure you are in the folder that includes `install.R`, `.s2i/environment`, `R scripts`, and `model file`...etc.
-
-If this folder is managed by `Git`, please commit all changes into the git.
+- Make sure you are in the folder that includes `install.R`, `Dockerfile`, `R scripts`, and `model file`.
 
-We will use `seldonio/seldon-core-s2i-r:0.1` as a base image, installing environment and packaging our model file into the target, `my-model-image`. You can use the following command to package the docker image:
+- If this folder is managed by `Git`, please commit all changes into the git.
```
Comment (jackklpan) [inline docs/model-deployment-language-wrapper-r.md:92]: Not required

File: docs/model-deployment-language-wrapper-python.md (hunk 1 above)
Comment (ggosiang) [reply docs/model-deployment-language-wrapper-python.md]: ok, updated

File: docs/model-deployment-language-wrapper-r.md (hunk 2 above)
Comment (ggosiang) [reply docs/model-deployment-language-wrapper-r.md]: ok, updated

**Comment by jackklpan** [comment]
LGTM

**Pull request merged**